app/views/posts/_post_info.html.erb

```diff
@@ -0,0 +1,20 @@
+<div class="post-container">
+  <h3><%= link_to post.title, user_post_path(user, post) %></h3>
+
+  <p class="post-content"><%= post.text %></p>
+
+  <p class="counter">
+    Comments: <%= post.comments.size %>,
+    Likes: <%= post.likes_counter.to_i %>
+  </p>
+</div>
+
+<% if defined?(show_comments) && show_comments && post.comments.present? %>
+<div class="comments-container">
+  <% comments.each do |comment| %>
+    <p>
+      <%= comment.user.name %>: <%= comment.text %>
+    </p>
+  <% end %>
+</div>
```

Comment:
- [ ] @mershark Great work displaying all the information on the user's post page, however, you are supposed to use the `most_recent_comments` post model method to display the five latest comments on this page.